doc/source/WorkingWithScenes/AnatomyOfACamera/index.md

```diff
@@ -0,0 +1,136 @@
+
+For depth of field, lens aperture is calculated with the formula: 
+
+```
+lens aperture = (focalLength × focalLengthWorldScale) / fStop
+```
+
+For example, assume a lens with a focalLength of 50mm, an fStop of 4, and a world space measured in centimeters (default for Alembic/USD). First, we need to scale the focal length from mm to cm. Using a focalLengthWorldScale of 0.1 yields `50mm × 0.1 = 0.5cm`. This is the lens aperture at its widest. Dividing by the fStop of 4 results in the stopped-down lens aperture of `0.5cm / 4 = 0.125cm`.
+
+
+## Cameras in the Scene ##
+
+
+### Camera data ###
+
+Within the <a href="../AnatomyOfAScene/index.html#scene-hierarchy">scene paradigm</a>, a camera, just like any other scene component, starts with a location in the scene hierarchy. To define a camera, the location must have the following data:
+
+- **Transform:** The vectors that define the position and orientation of the camera.
+- **Object:** A special camera object at the location. Instead of geometry, the object stores camera data, called parameters.
+    - **Parameters:** The crucial values that define a camera, such as the perspective type, field of view/aperture, and depth of field settings. If defined, a special kind of optional parameters, called **render overrides**, will supercede the scene's <a href="../AnatomyOfAScene/index.html#options">render options</a> during computation and rendering.<br>
+    ![Camera parameters in the Scene Inspector](images/interfaceCameraParameters.png)
+- **Sets:** A list of sets the location belongs to. By default, every camera is assigned to an automatic "Cameras" set, accessible in the API by the `__cameras` variable.<br>
+    ![Camera sets in the Scene Inspector](images/interfaceCameraSets.png)
+
+
+### Data flow ###
+
+For the most part, camera data follow the same rules as other scene data. The notable exception is in the handling of the scene's render settings.
+
+```eval_rst
+.. figure:: images/illustrationCameraDataFlow.png
+    :scale: 100%
+    :alt: Camera data flow in a node graph
+
+    **Figure 4.** The camera data flow through a node graph, with data passing between a Camera node, CameraTweaks node, and a StandardOptions node.
+```
+
+Each Camera node creates a camera, appearing in the scene hierarchy as its own location (by default called `/camera`). Each location will have its respective data. The locations and the data are passed to downstream scene nodes, like normal.
+
+A downstream CameraTweaks node can add new parameters and render options, or override/remove the existing parameters, of cameras and render options in the scene. Strictly speaking, there are no mandatory camera parameters, as Gaffer will supply fallback defaults.
```

Comment:
To me, "and render options" is implying that the CameraTweaks node can change data in the globals, but this is not the case. It can only change camera _parameters_. We need to make it clearer that parameters can override the globals.